wings/1.0/configuration.md

```diff
@@ -29,3 +30,70 @@ docker:
       username: "registryusername"
       password: "registrypassword"
 ```
+
+## Custom Network Interfaces
+
+You can change the network interface that Wings uses for all containers by editing the network name; it is by default set to `pterodactyl_nw`. For example, to enable Docker host mode change the network name to `host`.
+
+::: warning
+Changing network mode to `host` grants Pterodactyl direct access to all machine interfaces and panel users can bind to any IP or Port even if it's not allocated to their container. You will lose all benefits of Docker network isolation. It is not recommended for public installations that are hosting other users' servers.
+:::
+
+### Example of usage
+
+```yml
+docker:
+  network:
+    name: host
+    network_mode: host
+```
+
+After making changes, the following commands will stop the Wings, remove the Pterodactyl network, and start the Wings again. Run at your own risk.
+`systemctl stop wings && docker network rm pterodactyl_nw && systemctl start wings`
```

Comment:
Removing network will fail as long as any of the containers attached to it are running.